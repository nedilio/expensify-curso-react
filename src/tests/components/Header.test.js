import React from 'react';
import Header from '../../components/Header';
import reactShallowRenderer from 'react-test-renderer/shallow';

test('render header', () => {
  const renderer = new reactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
})