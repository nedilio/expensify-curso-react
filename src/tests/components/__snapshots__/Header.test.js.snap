// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <nav
    className="menu"
  >
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/create"
    >
      Create Expense
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/help"
    >
      Help
    </NavLink>
  </nav>
</header>
`;
